keystrokes/cypress/integration/game.spec.js

```diff
@@ -29,3 +30,29 @@ describe('Tests for the Game', () => {
 		});
 	});
 });
+
+describe('Tests for the game with a empty user', () => {
+	it('authenticates a different account', () => {
+		cy.logout();
```

Comment:
Still curious about this

Reply:
I add the logout because when i had the 2 different users there would sometimes be issues with some tests using the wrong user. I probably dont need to be as excessive about it, but i didnt want there to be any risk of it using the wrong account.